client/src/context/HomeContext/hooks.js

```diff
@@ -0,0 +1,104 @@
+const setChannelsHandler = dispatch => (channels) => {
+  dispatch({
+    type: 'SET_CHANNELS',
```

Comment:
@jaspreet57  naked strings could be converted into constant I think.

Reply:
done